api/src/test/java/com/github/streamshub/console/api/KafkaClustersResourceIT.java

```diff
@@ -786,66 +787,63 @@ void testDescribeClusterWithMetricsNotEnabled() {
 
     @ParameterizedTest
     @CsvSource({
-        "true",
-        "false"
+        "true, null, 3.6.0, 3.6.0",  // status is null due to paused reconciliation, fallback to spec
+        "false, 3.5.0, 3.6.0, 3.5.0" // status is present, use it
     })
-    void testPatchClusterReconciliationPaused(Boolean paused) {
-        whenRequesting(req -> req.get("{clusterId}", clusterId1))
-            .assertThat()
-            .statusCode(is(Status.OK.getStatusCode()))
-            .body("data.attributes.name", is("test-kafka1"))
-            .body("data.meta", not(hasKey("reconciliationPaused")));
+    void testPatchClusterReconciliationPaused(
```

Comment:
I wonder if these should be two separate tests. The original one that tests the PATCH with and without the `meta` to pause the Kafka reconciliation it important to keep.